node/src/chain_spec.rs

```diff
@@ -196,27 +164,21 @@ pub fn development_config(id: ParaId) -> ChainSpec {
 	properties.insert("tokenDecimals".into(), 11.into());
 	properties.insert("ss58Format".into(), 42.into());
 
-	ChainSpec::from_genesis(
-		// Name
-		"Parachain Eden Development",
-		// ID
-		"para_eden_dev",
-		ChainType::Development,
-		move || development_config_genesis(id),
-		Vec::new(),
-		None,
-		None,
-		None,
-		None,
+	ChainSpec::builder(
```

Comment:
```suggestion
	#[allow(deprecated)]
	ChainSpec::builder(
```

Reply:
the point here is not us use the deprecated function